bc/assets/templates/includes/footer.html

```diff
@@ -0,0 +1,122 @@
+              {% include "./footer_link.html" with href="https://free.law/data-consulting" text="Data Services and Consulting" %}
+              {% include "./footer_link.html" with href="https://github.com/freelawproject" text="Other Projects on Github" %}
+            </div>
+          </div>
+
+          <div class="w-full">
+            <div class="pb-3">
+              <H5>little cases</H5>
+              {% include "./footer_link.html" with href="#" text="Build your own bot" %}
+            </div>
+            <div class="pb-3">
+              <H5>More about flp</H5>
+              {% include "./footer_link.html" with href="https://free.law/donate/" text="Donate Now" %}
+              {% include "./footer_link.html" with href="#" text="Follow the FLP Blog" %}
+              {% include "./footer_link.html" with href="https://free.law/vulnerability-disclosure-policy" text="Vulnerability Disclosure Policy" %}
+              {% include "./footer_link.html" with href="https://free.law/contact/" text="Contact Us" %}
+            </div>
+          </div>
+      </section>
+    </div>
+</section>
+
+<section class="border-t border-gray-300 mt-0 sm:mt-8 md:mt-16">
+    <div class="mx-auto px-4 sm:px-6 md:px-10">
+        <div class="py-12 flex flex-wrap justify-between" id="colophon">
+            <div class="w-full sm:w-3/5 md:w-1/2 lg:w-2/5 text-gray-400 content-center">
+                <p class="text-sm">
+                © {% now "Y" %} Free Law Project. Content licensed under a Creative Commons
+                BY-ND international 4.0, license, except where indicated.
+                </p>
+            </div>
+            <div class="w-full sm:w-2/5 md:w-1/2 ">
+                <div class="flex flex-row justify-center pt-6 sm:pt-0 md:justify-end sm:justify-end gap-1">
+                  <div class="hidden md:flex items-center mr-4">
+                    <p class="text-gray-400"> Follow us: </p>
+                  </div>
+                  <a href="https://twitter.com/freelawproject" class="rounded-full bg-gray-500 p-3">
+                    <svg
+                      xmlns="http://www.w3.org/2000/svg"
+                      class="h-5 w-5"
+                      fill="white"
+                      viewBox="0 0 24 24">
+                      <path
+                        d="M24 4.557c-.883.392-1.832.656-2.828.775 1.017-.609 1.798-1.574 2.165-2.724-.951.564-2.005.974-3.127 1.195-.897-.957-2.178-1.555-3.594-1.555-3.179 0-5.515 2.966-4.797 6.045-4.091-.205-7.719-2.165-10.148-5.144-1.29 2.213-.669 5.108 1.523 6.574-.806-.026-1.566-.247-2.229-.616-.054 2.281 1.581 4.415 3.949 4.89-.693.188-1.452.232-2.224.084.626 1.956 2.444 3.379 4.6 3.419-2.07 1.623-4.678 2.348-7.29 2.04 2.179 1.397 4.768 2.212 7.548 2.212 9.142 0 14.307-7.721 13.995-14.646.962-.695 1.797-1.562 2.457-2.549z" />
+                    </svg>
+                  </a>
+                  <a rel="me" href="https://law.builders/@flp" class="rounded-full bg-gray-500 p-3">
```

Comment:
Let's put mastodon first and Twitter second.

Reply:
Alright